### DISCUSSION TITLE
Why is a TLS runtime required?

### DISCUSSION THREAD

**Discussion by andrewbaxter** [SeaQL/sea-orm#1003] [Q&A] [answered]
I'd like to use SeaORM with sqlite, but the build fails if I don't enable a TLS feature. The documentation only says
> You have to choose one

with no further explanation.

**Comment by andrewbaxter** [comment, marked as answer]
Ah, this is a requirement forwarded from SQLx it looks like.  It would be nice to note that in the documentation.  It seems it's not intentional and there's work being done to remove this requirement: https://github.com/launchbadge/sqlx/issues/1750

**Reply by billy1624** [reply]
Hey @andrewbaxter, welcome! Correct, this limitation actually comes from SQLx. We'd like to get rid of this constraint once SQLx managed to compile without depending on runtime.
